ext/dapr-ext-workflow/README.rst

```diff
@@ -16,6 +16,490 @@ Installation
 
     pip install dapr-ext-workflow
 
+Async authoring (experimental)
+------------------------------
+
+This package supports authoring workflows with ``async def`` in addition to the existing generator-based orchestrators.
+
+- Register async workflows using ``WorkflowRuntime.workflow`` (auto-detects coroutine) or ``async_workflow`` / ``register_async_workflow``.
+- Use ``AsyncWorkflowContext`` for deterministic operations:
+
+  - Activities: ``await ctx.call_activity(activity_fn, input=...)``
+  - Child workflows: ``await ctx.call_child_workflow(workflow_fn, input=...)``
+  - Timers: ``await ctx.create_timer(seconds|timedelta)``
+  - External events: ``await ctx.wait_for_external_event(name)``
+  - Concurrency: ``await ctx.when_all([...])``, ``await ctx.when_any([...])``
+  - Deterministic utils: ``ctx.now()``, ``ctx.random()``, ``ctx.uuid4()``, ``ctx.new_guid()``, ``ctx.random_string(length)``
+
+Interceptors (client/runtime/outbound)
+--------------------------------------
+
+Interceptors provide a simple, composable way to apply cross-cutting behavior with a single
+enter/exit per call. There are three types:
+
+- Client interceptors: wrap outbound scheduling from the client (schedule_new_workflow).
```

Comment:
understand these a bit better when referencing the example code. Can you update these a bit to be clearer pls? You have a client interceptor but have outbound in its description but then also have outbound interceptors. Are there inbound interceptors? if not, then why not?